app/src/main/java/app/revanced/manager/ui/viewmodel/InstallerViewModel.kt

```diff
@@ -215,6 +220,14 @@ class InstallerViewModel(
         }
     }
 
+    fun reinstall() = viewModelScope.launch {
+        pm.getPackageInfo(outputFile)?.packageName?.let { pm.uninstallPackage(it) }
+            ?: throw Exception("Failed to load application info")
+
+        pm.uninstallPackage(packageName)
```

Comment:
`pm.uninstallPackage` is run twice here
Also is the exception intended here? It's not handled so the whole app will crash if the app isnt installed.